src/scripts/extract_items_from_content_store.python

```diff
@@ -0,0 +1,44 @@
+import csv
+import gzip
+import sys
+
+# The purpose of this script is to take a list of URLs and extract the
+# preprocessed content items for those urls from the store into another csv file
+# that can be passed to the report runner. This makes it quicker to test new and
+# existing reports where you already know one URL with an issue they should
+# handle.
+
+gzipped_content_store_file_path = "downloads/preprocessed_content_store_210722.csv.gz"
+target_file_path = "downloads/abridged_content_store_210722.csv.gz"
```

Comment:
Do we want to leave these hard-coded in here?
In fact could this script be it's own PR separate from the table relationship report?

Reply:
If leaving hard-coded, perhaps call out they are examples and will need to be updated, but then I suppose that's also self explanatory